Crash report 3.0.2: IllegalStateException in ReaderPostDetailFragment

```
java.lang.RuntimeException: An error occured while executing doInBackground()
       at android.os.AsyncTask$3.done(AsyncTask.java:300)
       at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
       at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
       at java.util.concurrent.FutureTask.run(FutureTask.java:242)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
Caused by: java.lang.IllegalStateException: Fragment ReaderPostDetailFragment{43437d18} not attached to Activity
       at android.app.Fragment.getResources(Fragment.java:767)
       at org.wordpress.android.ui.reader.ReaderPostDetailFragment.getFullSizeImageWidth(ReaderPostDetailFragment.java:978)
       at org.wordpress.android.ui.reader.ReaderPostDetailFragment.getPostHtml(ReaderPostDetailFragment.java:1014)
       at org.wordpress.android.ui.reader.ReaderPostDetailFragment.access$2800(ReaderPostDetailFragment.java:65)
       at org.wordpress.android.ui.reader.ReaderPostDetailFragment$ShowPostTask.doInBackground(ReaderPostDetailFragment.java:1209)
       at org.wordpress.android.ui.reader.ReaderPostDetailFragment$ShowPostTask.doInBackground(ReaderPostDetailFragment.java:1156)
       at android.os.AsyncTask$2.call(AsyncTask.java:288)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
```

--- a/WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.java
@@ -219,12 +219,13 @@ public class ReaderPostDetailFragment extends Fragment
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         final View view = inflater.inflate(R.layout.reader_fragment_post_detail, container, false);
+        final Context context = container.getContext();
 
         // locate & init listView
         mListView = (WPListView) view.findViewById(android.R.id.list);
         if (isFullScreenSupported()) {
             mListView.setOnScrollDirectionListener(this);
-            ReaderUtils.addListViewHeader(mListView, DisplayUtils.getActionBarHeight(container.getContext()));
+            ReaderUtils.addListViewHeader(mListView, DisplayUtils.getActionBarHeight(context));
         }
 
         // add post detail as header to listView - must be done before setting adapter
@@ -235,7 +236,7 @@ public class ReaderPostDetailFragment extends Fragment
         // progress bar appears when loading new comments
         mCommentFooter = (ViewGroup) inflater.inflate(R.layout.reader_footer_progress, mListView, false);
         mCommentFooter.setVisibility(View.GONE);
-        mCommentFooter.setBackgroundColor(getResources().getColor(R.color.grey_extra_light));
+        mCommentFooter.setBackgroundColor(context.getResources().getColor(R.color.grey_extra_light));
         mProgressFooter = (ProgressBar) mCommentFooter.findViewById(R.id.progress_footer);
         mProgressFooter.setVisibility(View.INVISIBLE);
         mListView.addFooterView(mCommentFooter);
@@ -1018,10 +1019,10 @@ public class ReaderPostDetailFragment extends Fragment
      */
     private int getFullSizeImageWidth(Context context) {
         int displayWidth = DisplayUtils.getDisplayPixelWidth(context);
-        int marginWidth = getResources().getDimensionPixelOffset(R.dimen.reader_list_margin);
+        int marginWidth = context.getResources().getDimensionPixelOffset(R.dimen.reader_list_margin);
         int imageWidth = displayWidth - (marginWidth * 2);
         if (hasStaticMenuDrawer()) {
-            int drawerWidth = getResources().getDimensionPixelOffset(R.dimen.menu_drawer_width);
+            int drawerWidth = context.getResources().getDimensionPixelOffset(R.dimen.menu_drawer_width);
             imageWidth -= drawerWidth;
         }
         return imageWidth;

